tests/app/googleanalytics/test_pixels.py

```diff
@@ -1,77 +1,60 @@
-from urllib.parse import quote_plus
+import os
+from unittest.mock import patch
 
-from flask import current_app
+import pytest
 
-from app.googleanalytics.pixels import build_ga_pixel_url
+from app.googleanalytics.pixels import build_dynamic_ga4_pixel_tracking_url
 
 
-def test_build_ga_pixel_url_contains_expected_parameters(
+@pytest.mark.parametrize(
+    'env, expected_domain',
+    [
+        ('test', 'https://test-api.va.gov/vanotify/'),
+        ('development', 'https://dev-api.va.gov/vanotify/'),
+        ('perf', 'https://sandbox-api.va.gov/vanotify/'),
+        ('staging', 'https://staging-api.va.gov/vanotify/'),
+        ('prod', 'https://api.va.gov/vanotify/'),
+    ],
+)
+def test_build_dynamic_ga4_pixel_tracking_url_correct_domain_for_environment(
```

Comment:
As this is a new test, please use the new [test naming convention](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Process/testing_guide.md).

Reply:
Updated.